doc/benchmarking.md

```diff
@@ -2,54 +2,46 @@ Benchmarking
 ============
 
 Bitcoin Core has an internal benchmarking framework, with benchmarks
-for cryptographic algorithms (e.g. SHA1, SHA256, SHA512, RIPEMD160), as well as the rolling bloom filter.
+for cryptographic algorithms (e.g. SHA1, SHA256, SHA512, RIPEMD160, Poly1305, ChaCha20), rolling bloom filter, coins selection,
+thread queue, wallet balance.
 
 Running
 ---------------------
+
+For benchmarks purposes you only need to compile `bitcoin_bench`. Beware to configure without `--enable-debug` as it impacts
+benchmarking
```

Comment:
Could you describe how/why omitting the `--enable-debug` argument would impact benchmarking?

Nit: missing punctuation at end of sentence.

Reply:
I would say main point is lock analysis, updated